src/config/Modules.yml

```diff
@@ -50,3 +50,4 @@ modules:
     - name: validation
       depends: ['array_ops', 'regress']
     - name: stemmer
+    - name: cols_vec
```

Comment:
I'm not convinced that we need a new module for this functionality. IMO this is better suited for the `utilities` module as a separate file.

Reply:
Discussed it with @fmcquillan99 and I will move it under Utilities module in next commit.